api/types/servers.go

```diff
@@ -25,9 +27,23 @@ type Dns struct {
 }
 
 type ScriptChar struct {
-	ID              string   `json:"id" header:"ID"`
-	Type            string   `json:"type" header:"TYPE"`
-	ParameterValues struct{} `json:"parameter_values" header:"PARAMETER_VALUES"`
-	TemplateID      string   `json:"template_id" header:"TEMPLATE_ID"`
-	ScriptID        string   `json:"script_id" header:"SCRIPT_ID"`
+	ResourceType    string           `json:"resource_type" header:"RESOURCE_TYPE" show:"noshow,nolist"`
+	ID              string           `json:"id" header:"ID"`
+	Type            string           `json:"type" header:"TYPE"`
+	ParameterValues *json.RawMessage `json:"parameter_values" header:"PARAMETER_VALUES"`
+	TemplateID      string           `json:"template_id" header:"TEMPLATE_ID"`
+	ScriptID        string           `json:"script_id" header:"SCRIPT_ID"`
+	ExecutionOrder  int              `json:"execution_order" header:"EXECUTION_ORDER" show:"noshow,nolist"`
+}
+
+type ScriptCharResponse struct {
```

Comment:
The response for an execution is actually an event, I think you should reuse the event type for it...

Reply:
Done